Update cross-origin isolation related articles (#5101)
GoogleChrome/web.dev: src/site/content/en/secure/cross-origin-isolation-guide/index.md

precision](https://developer.chrome.com/blog/cross-origin-isolated-hr-timers/),
or the JS Self-Profiling API.

If you intend to enable cross-origin isolation, evaluate the impact this will

> have on other cross-origin resources on your website, e.g. ad placements.

Reviewer: ```suggestion
have on other cross-origin resources on your website, such as ad placements.
```

{% endAside %}

{% Aside 'caution' %}

Enabling cross-origin isolation will block the loading of cross-origin resources
that you don't explicitly opt-in, and it will prevent your top-level document
from being able to communicate with popup windows.

We've been exploring ways to deploy `Cross-Origin-Resource-Policy` at scale, as
cross-origin isolation requires all subresources to explicitly opt-in. And we
have come up with the idea of going in the opposite direction: [a new COEP
"credentialless" mode](https://github.com/mikewest/credentiallessness/) that
allows loading resources without the CORP header by stripping all their

> credentials. We are working out how it should work in detail, but we hope this
> will lighten your burden of making sure the subresources are sending the
> `Cross-Origin-Resource-Policy` header.

Reviewer: ```suggestion
credentials. We are figuring out the details of how it should work, but we hope this
will lighten your burden of making sure the subresources are sending the
`Cross-Origin-Resource-Policy` header.
```

Also, it's known that the `Cross-Origin-Opener-Policy: same-origin` header will
break integrations that require cross-origin window interactions such as OAuth
and payments. To mitigate this problem, we are exploring [relaxing the
condition](https://github.com/whatwg/html/issues/6364) to enable cross-origin
isolation to `Cross-Origin-Opener-Policy: same-origin-allow-popups`. This way
the communication with the window opened by itself will be possible.

If you want to enable cross-origin isolation but are blocked by these
challenges, we recommend [registering for an origin
trial](https://developer.chrome.com/blog/enabling-shared-array-buffer/#origin-trial)

> and wait until the new modes are available. We are not planning to terminate the

Reviewer: ```suggestion
and waiting until the new modes are available. We are not planning to terminate the
```
